Sources/Orbit/Components/AlertInline.swift

```diff
@@ -0,0 +1,216 @@
+import SwiftUI
+
+/// Breaks the main user flow to present information.
+///
+/// There are times when just simple information isn’t enough and the user needs
+/// to take additional steps to resolve the problem or get additional details.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/alert/)
+/// - Important: Component expands horizontally unless prevented by `fixedSize` or `idealSize` modifier.
+public struct AlertInline<Icon: View, Button: View>: View {
+
+    let title: String
+    let isSubtle: Bool
+    @ViewBuilder let button: Button
+    @ViewBuilder let icon: Icon
+
+    public var body: some View {
+        AlertContent(title: title, description: "", isInline: true, isSubtle: isSubtle) {
+            EmptyView()
+        } icon: {
+            icon
+        } buttons: {
+            button
+        }
+    }
+}
+
+public extension AlertInline {
+
+    /// Creates Orbit AlertInline component.
+    init(
+        _ title: String = "",
+        icon: Icon.Symbol? = nil,
+        isSubtle: Bool = false,
+        @AlertInlineButtonsBuilder button: () -> Button = { EmptyView() }
+    ) where Icon == Orbit.Icon {
+        self.init(title, isSubtle: isSubtle, button: button) {
+            Icon(icon)
+        }
+    }
+
+    /// Creates Orbit AlertInline component with custom content and icon.
+    init(
+        _ title: String = "",
+        isSubtle: Bool = false,
+        @AlertInlineButtonsBuilder button: () -> Button,
+        @ViewBuilder icon: () -> Icon
+    ) {
+        self.title = title
+        self.isSubtle = isSubtle
+        self.button = button()
+        self.icon = icon()
+    }
+}
+
+// MARK: - Types
+
+/// A builder that constructs buttons for the ``AlertInline`` component.
+@resultBuilder
+public enum AlertInlineButtonsBuilder {
+
+    public static func buildBlock(_ emptyView: EmptyView) -> EmptyView {
+        emptyView
+    }
+
+    public static func buildBlock(_ button: some View) -> some View {
+        button
+            .suppressButtonStyle()
+            .buttonStyle(OrbitButtonLinkButtonStyle(type: .status(nil), icon: {}, trailingIcon: {}))
```

Comment:
We could add { EmptyView() } as default values for both `OrbitButtonLinkButtonStyle` and `OrbitButtonStyle`

Reply:
These were replaced with `AlertButtonStyle` and `AlertInlineButtonStyle`, which have no icons.